### DISCUSSION TITLE
Return codes in the json results shown as -1 and 9?

### DISCUSSION THREAD

**Discussion by mbrondogutierrez** [fortio/fortio#479] [Q&A] [answered]
We ran a test and I'm seeing a number of requests in the following buckets and wanted to know what they mean. two examples are below.

 "RetCodes": {
    "200": 206661,
    "-1": 226768
  },

  "RetCodes": {
    "9": 383806,
    "200": 504092,
    "-1": 129627
  },

**Comment by ldemailly** [comment, marked as answer]
-1 is connection error. -9 I am a bit curious about. All errors should be printed in details in stderr, the table is just the summary. Can you share a sample of full output (minus any identifiable information)? (adding "sample" as you seem to be getting a lot of errors which is not expected either)

**Comment by ldemailly** [comment]
@mbrondogutierrez any more details on the "9"?

**Reply by ldemailly** [reply]
@mbrondogutierrez would love to hear more - please comment or open an issue if you see these 9 again (and how)
